docs/Consensus-Protocols/IBFT.md

```diff
@@ -132,9 +132,15 @@ The JSON-RPC methods to add or remove validators are:
 * [ibft_proposeValidatorVote](../Reference/Pantheon-API-Methods.md#ibft_proposeValidatorVote)
 * [ibft_discardValidatorVote](../Reference/Pantheon-API-Methods.md#ibft_discardValidatorVote)
 
-Use [ibft_getSignerMetrics](../Reference/Pantheon-API-Methods.md#ibft_getsignermetrics) to view signer metrics for a specified block range.
+!!! important
+    Adding or removing a validator is subject to agreement by existing validators. Meaning the command to add or remove a validator must
+    be executed on the majority of validators (2/3) to take effect.
```

Comment:
Only a majority of validators is required, not 2/3rds.  It's exactly the same code as for Clique.  `number_of_validators / 2 + 1` is required (ie a clear majority).

Reply:
I discussed that with @ajsutton and it looks like:
- super majority (2/3rd = ~66%) is required to validate a block
- majority (number_of_validators / 2 + 1) is only required to add/remove validators

See highlighted parts in https://media.consensys.net/scaling-consensus-for-enterprise-explaining-the-ibft-algorithm-ba86182ea668

Reply:
Thanks, this clears things up. @ajsutton does Clique also require  the super-majority to validate a block?

Reply:
Feedback from @ajsutton, "No clique proposers just create a block and there are no sign offs."